action-server/actions/self_isolation_form.py

```diff
@@ -0,0 +1,58 @@
+from typing import Any, Dict, List, Text, Union
+
+from rasa_sdk import Tracker
+from rasa_sdk.events import EventType
+from rasa_sdk.executor import CollectingDispatcher
+from rasa_sdk.forms import FormAction
+
+LIVES_ALONE_SLOT = "lives_alone"
+
+
+class SelfIsolationForm(FormAction):
+    def name(self) -> Text:
+
+        return "self_isolation_form"
+
+    ## override to play initial message
+    async def _activate_if_required(
+        self,
+        dispatcher: "CollectingDispatcher",
+        tracker: "Tracker",
+        domain: Dict[Text, Any],
+    ) -> List[EventType]:
+        if tracker.get_slot(LIVES_ALONE_SLOT) is None:
+            dispatcher.utter_message(template="utter_symptoms_self_isolate")
+
+        return await super()._activate_if_required(dispatcher, tracker, domain)
```

Comment:
This is always invoked in the form run method:
`events = await self._activate_if_required(dispatcher, tracker, domain)`

Your message will be played every time the form is invoked, which is not what we want.